test/utils/InstanceTest.sol

```diff
@@ -42,6 +42,8 @@ abstract contract InstanceTest is HyperdriveTest {
         bool enableShareDeposits;
         bool enableBaseWithdraws;
         bool enableShareWithdraws;
+        // TODO: Add variants of this for the other "enable" states.
```

Comment:
does this TODO need to be addressed?

Reply:
No, I left this so that we think about it in the future. That's why I left it as a TODO FWIW. Anytime there is something that I want to do in this PR, I make it a `FIXME`.